localstack/utils/docker_utils.py

```diff
@@ -1508,6 +1569,26 @@ def has_docker(self) -> bool:
         except APIError:
             return False
 
+    def remove_image(self, image: str, force: bool = True):
+        LOG.debug("Removing image %s %s", image, "(forced)" if force else "")
+        try:
+            self.client().images.remove(image=image, force=force)
+        except ImageNotFound:
+            if not force:
+                raise NoSuchImage(image)
+
+    def commit(
+        self,
+        container_name_or_id: str,
+        image_name: str,
+        image_tag: str,
+    ):
+        LOG.debug(
+            "Creating image from container %s as %s:%s", container_name_or_id, image_name, image_tag
+        )
+        container = self.client().containers.get(container_name_or_id)
```

Comment:
Please add exception handling here if the target container does not exist, including a test for the behavior.
Also wrapping the whole thing to map API errors to container exceptions would be nice.